Document/0x06a-Platform-Overview.md

```diff
@@ -48,35 +48,33 @@ Signing application code in iOS is different than in Android. In the latter you
 At least two scenarios exists where you can install an application without the App Store:
 
 1. Via Enterprise Mobile Device Management. This requires the company to have company-wide certificate signed by Apple.
-2. Via side-loading - i.e. by signing the app with a developer's certificate and installing it on the device via Xcode. There is an upper limit of number of devices that can be used with the same certificate.
+2. Via side-loading, i.e. by signing an app with the developer's certificate and installing it on the device via Xcode. There is a top limit of the number of devices that can be used with the same certificate.

```

Comment:
> There is an upper/a top limit

I think "upper limit" is actually a commonly used phrase in English, whereas "top limit" is not. I think "upper limit" is better.

> by signing an app with a/the developer's

Is it really *the* developer's certificate? Is there only one developer with one certificate, or is any certificate of any developer OK?